pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -0,0 +1,133 @@
+package preflight
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/version"
+	"github.com/code-ready/crc/pkg/os/windows/powershell"
+)
+
+var (
+	// https://docs.microsoft.com/en-us/windows/win32/taskschd/daily-trigger-example--xml-
+	daemonTaskTemplate = `<?xml version="1.0" encoding="UTF-16"?>
+<Task version="1.3" xmlns="http://schemas.microsoft.com/windows/2004/02/mit/task">
+  <RegistrationInfo>
+    <Description>Run crc daemon as a task</Description>
+    <Version>%s</Version>
+  </RegistrationInfo>
+  <Settings>
+    <DisallowStartIfOnBatteries>false</DisallowStartIfOnBatteries>
+    <StopIfGoingOnBatteries>false</StopIfGoingOnBatteries>
+    <Hidden>true</Hidden>
+    <MultipleInstancesPolicy>IgnoreNew</MultipleInstancesPolicy>
+    <IdleSettings>
+      <Duration>PT10M</Duration>
+      <WaitTimeout>PT1H</WaitTimeout>
+      <StopOnIdleEnd>true</StopOnIdleEnd>
+      <RestartOnIdle>false</RestartOnIdle>
+    </IdleSettings>
+    <UseUnifiedSchedulingEngine>true</UseUnifiedSchedulingEngine>
+  </Settings>
+  <Triggers />
+  <Actions Context="Author">
+    <Exec>
+      <Command>powershell.exe</Command>
+      <Arguments>-WindowStyle Hidden -Command %s</Arguments>
```

Comment:
If what %s is substituted for contains `<`, `>`, `&`, ... this will fail badly. If you want to be 100% safe, you probably need https://pkg.go.dev/encoding/xml#EscapeText

Reply:
Done